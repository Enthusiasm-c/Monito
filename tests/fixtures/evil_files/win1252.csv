Товар;Цена;Описание
"Смартфон ""iPhone""";99 999₽;Флагманский смартфон—отличное качество
Ноутбук №1;150 000р.;Профессиональный ноутбук—высокая производительность
"Планшет ""iPad""";59 999₽;Планшет для работы—легкий и мощный
Наушники;25 000р.;Беспроводные наушники—premium качество
