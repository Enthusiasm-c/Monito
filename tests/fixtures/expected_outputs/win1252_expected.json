{
  "filename": "win1252.csv",
  "expected_rows_in": 4,
  "expected_rows_out": 4,
  "encoding": "utf-8",
  "delimiter": ";",
  "challenges": [
    "Encoding issues",
    "Special characters",
    "European CSV format"
  ]
}